### PR TITLE
Corner radius and elevation levels fixed

### LINKED ISSUE #2468: Card Corner Radius is specified 0dp in dimens.xml file
**Actual Behaviour**

The card corner radius is 0 in the dimensions file

**Expected Behaviour**

This should not be the case

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #2468

### CODE REVIEW

**Comment by dreadpool2** [comment]
@iamareebjamal Please review

File: android/app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -15,8 +15,12 @@
...
 
     <!--card view specifics -->
-    <dimen name="card_elevation">2dp</dimen>
-    <dimen name="card_corner_radius">0dp</dimen>
+    <dimen name="card_small_elevation">2dp</dimen>
+    <dimen name="card_medium_elevation">4dp</dimen>
+    <dimen name="card_large_elevation">6dp</dimen>
+    <dimen name="card_corner_small_radius">2dp</dimen>
+    <dimen name="card_corner_medium_radius">4dp</dimen>
+    <dimen name="card_corner_large_radius">6dp</dimen>
```
Comment (iamareebjamal) [inline android/app/src/main/res/values/dimens.xml]: No need to create so many dimens. I think just fixing the default value is enough

Comment (dreadpool2) [reply android/app/src/main/res/values/dimens.xml]: Okay 👍

**Comment by dreadpool2** [comment]
Updated the PR

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**